src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+const convertToObject = (sourceString) => {
+  const splittedSource = sourceString.split(';');
```

Comment:
The variable name 'splittedSource' could be more descriptive. Consider renaming it to reflect that it contains style declarations, such as 'styleDeclarations' or 'styleEntries'.